doc_src/cmds/complete.rst

```diff
@@ -36,8 +36,8 @@ The following options are available:
 **-l** or **--long-option** *LONG_OPTION*
     Adds a GNU-style long option to the completions list.
 
-**-o** or **--old-option** *LONG_OPTION*
-    Adds an old-style long option to the completions list (see below for details).
+**-o** or **--old-option** *OPTION*
+    Adds an old-style short or long options (see below for details).
```

Comment:
Broken plural - "Adds an options"

Reply:
How about:

```
Adds an old-style single-dash option (see below for details
```

?

Reply:
Thanks for catching that. Given that the other examples (short, GNU-style long) specify short and long, I figured it makes sense to specify that here, too.